src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [tablePeople, setTablePeople] = useState<Person[]>([]);
+  const [hasLoading, setHasloading] = useState<boolean>(true);
+  const [hasError, setHasError] = useState<boolean>(false);
+
+  const hasTablePeople = !!tablePeople.length && !hasLoading && !hasError;
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const fetchedPeople = await getPeople();
+
+        setTablePeople(fetchedPeople);
+      } catch (error) {
+        setHasError(true);
+      } finally {
+        setHasloading(false);
```

Comment:
Ensure consistency in naming by using `setHasLoading` instead of `setHasloading` to avoid confusion.